docs/index.asciidoc

```diff
@@ -107,15 +107,15 @@ filter {
       },
       {
         id => "local-users"
-        query => "select firstname, lastname from users WHERE userid = :id"
-        parameters => {id => "[loggedin_userid]"}
-        target => "user" <4>
+        query => "select firstname, lastname from users WHERE userid = ?"
+        prepared_parameters => ["[loggedin_userid]"] <4>
```

Comment:
How about having an example with > 1 `prepared_parameters`? There could be confusion as to what the correct syntax here (is it `["[param1, param2]"]` or `["[param1]", "[param2]"]`? Or does it not matter?) It might also help to reiterate here that the order of the parameters in `prepared_parameters` is important